components/playlist/browser/resources/components/player.tsx

```diff
@@ -56,42 +77,68 @@ const PlayerContainer = styled.div<{ backgroundUrl?: string }>`
     opacity: 0.2;
     z-index: -1;
   }
+
+  @media ${playerTypes.miniPlayer} {
+    display: flex;
+    flex-direction: row;
+    gap: ${spacing.xl};
+  }
 `
 
-const ControlsContainer = styled.div`
+const ControlsContainer = styled.div<{ mouseHovered: boolean }>`
+  display: flex;
   position: relative;
-  ${playlistControlsAreaHeight}
-  height: var(--player-controls-area-height);
+  @media ${playerTypes.normalPlayer} {
+    ${p =>
+      !p.mouseHovered &&
+      css`
+        visibility: hidden;
+      `};
+    position: absolute;
+    bottom: 0;
+    z-index: 1;
+    width: 100%;
 
-  display: flex;
-  flex-direction: column;
-  padding: ${spacing['2XL']} ${spacing.xl};
-  gap: 8px;
-  flex-shrink: 0;
-  justify-content: center;
+    flex-direction: column;
+    padding: ${spacing['2XL']} ${spacing.xl};
+    gap: 8px;
+    flex-shrink: 0;
+    justify-content: center;
+    box-sizing: border-box;
+    height: var(--player-controls-area-height);
+    --leo-icon-color: ${color.white};
+    --seeker-progress-background: color-mix(
+      in srgb,
```

Comment:
Wow, has this finally landed in Chrome?! Awesome, I've been waiting for years

Reply:
Yeah, I think so. I saw the opacity was applied

https://caniuse.com/?search=color-mix

I remember you mentioning that 👍